.github/workflows/docs.yml

```
@@ -0,0 +1,29 @@
+name: Docs
+
+on:
+  push:
+    branches: [main]
+
```

```suggestion
permissions:
  contents: read
  pages: write
  id-token: write

concurrency:
  group: "pages"
  cancel-in-progress: true

```
This information is required, see https://github.com/actions/starter-workflows/blob/main/pages/static.yml